webCommander/centralHub/spotify/spotifyAPI.py

```diff
@@ -47,14 +55,25 @@ def spotify_authenticate():
     headers = {"Authorization": f"Bearer {token}"}
     return headers
 
+def get_spotify_artist_by_id(artist_id):
+    '''
+    Authenticate then search an artist based on an artist name. Searches for the name if id is not provided
+    '''
+
+    url = SPOTIFY_BASE_URL + SPOTIFY_ARTISTS + artist_id
+
+    response = requests.get(url, headers=spotify_authenticate())
+    return response.text
```

Comment:
no `response.json()`?

Reply:
as per call, this change you definitely will need for the artist submit in view!